insurance-app/README.md

```diff
@@ -0,0 +1,119 @@
+Instead we suggest that users store a fingerprint (SHA-256 hash) of the incident description in the ledger and store the actual description of the incident in a separate database, that the processor then fetches the description from.
+This should allow for compliance with the relevant regulations.
+
+## Build and run this sample
+
+### ACL app
+
+- Ensure you have ACL running
+  - To run ACL locally in virtual mode:
+    - `git clone https://github.com/microsoft/tpal`
+    - build with `mkdir build && cd build && CC="clang-15" CXX="clang++-15" cmake -GNinja -DCOMPILE_TARGET=virtual .. && ninja`
+    - from the build directory run with `PLATFORM=virtual ../tests/start_network.sh /path/to/ccf_virtual/`
+- Build the bundle
+  - In `<repository-root>/insurance-app/acl-app`
+	- Install dependencies: `npm install .`
+	- Build the bundle: `npm run build`
+- If using a local TPAL installation, test it is working by running the unit test after building the bundle
+  - In `<repository-root>/insurance-app/acl-app`
+  - `npm run build && python scripts/local-tpal-unit-test.py --bundle dist/bundle.json --tpal-tests-directory <tpal>/tests/ --sandbox-common <tpal>/build/workspace/sandbox_common --setup --add-roles`
+  - This requires that the relevant packages from tpal's tests have been installed: `pip install -r <tpal>/tests/requirements.text`
+	- This unit test uses a statically configured policy and attested keys to do an end-to-end test.
+
+
+### C-ACI container
+
+- In `<repository-root>/insurance-app/c-aci`
+- Ensure that you have logged into the azure cli via `az login`
+- Ensure you have an azure container repository set up
+  - https://learn.microsoft.com/en-us/azure/container-registry/container-registry-get-started-azure-cli
+- Ensure that you have a hugging face token to download the model
+  - Get one at https://huggingface.com
+- Build and push to the acr the attestation sidecar
+  - `git clone https://github.com/microsoft/confidential-sidecar-containers.git`
+  - `cd confidential-sidecar-containers`
+  - `docker build -t <acr-name>.azurecr.io/attestation-sidecar -f docker/attestation-container/Dockerfile .`
+  - `docker login -u 00000000-0000-0000-0000-000000000000 -p $(az acr login --name <acr-name> --expose-token --output tsv --query accessToken) <acr-name>.azurecr.io`
+  - `docker push <acr-name>.azurecr.io/attestation-sidecar`
+- Update `<repository-root>/insurance-app/c-aci/env.sh`
+- Prepare image and arm-template
+  - `HUGGINGFACE_TOKEN=<hugging_face_token> bash prepare-deploy-aci.sh`
+	- Copy the hash in the final line of output for the policy
+- Deploy to c-aci
+  - `bash deploy-aci.sh`
+
+Note: This sample provides ssh access to the container for debugging and exploration.
+Production use should remove this and in the `arm-template.json` directly run the python processor.
+
+### Testing the sample against a local TPAL 
```

Comment:
@cjen1-msft please check with the team for the official name, but that's probably "Programmable ACL" instead.